From: Martin Quinson
Date: Sun, 4 Jan 2015 20:49:23 +0100
Subject: rawlua

use the right version of lua when not using luajit
When available, we use the luajit for performance. But when this is not
available (as on hurd mipsel and other architecture), we fallback to the raw
lua implementation.
Upstream embeeds its own copy of lua implementation for that case, but we prune
it and prefer the system version of lua. This patch is here to use the
correct name of the lua library in that case.

Forwarded: not-needed
---
 src/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index b308ad2..37dd3f9 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -261,9 +261,9 @@ if(LUA_LIBRARY AND LUA_INCLUDE_DIR)
 endif (LUA_LIBRARY AND LUA_INCLUDE_DIR)
 
 if(NOT USE_LUAJIT)
-	message (STATUS "LuaJIT not found, using bundled Lua.")
-	set(LUA_INCLUDE_DIR "${PROJECT_SOURCE_DIR}/lua/src")
-	set(LUA_LIBRARY "lua")
+	message (STATUS "LuaJIT not found, using the version of Lua provided by the system.")
+	set(LUA_INCLUDE_DIR "/usr/include/lua5.1/")
+	set(LUA_LIBRARY "lua5.1")
 endif(NOT USE_LUAJIT)
 
 mark_as_advanced(LUA_LIBRARY)
